demo/apiExamples.md

```diff
@@ -0,0 +1,97 @@
+
+## API Examples
+
+### Basic
+
+<div class="twoColDemoRow">
+  <div>
+    <div class="exampleWrapper">
+      <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
+      <!-- The below content is automatically added from ./../../apiExamples/basic.html -->
+      <auro-table
+        columnHeaders='["","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]'
+        componentData='[
+        {"": "Dance class", "Monday": "5:00pm","Wednesday": "5:00pm" },
+        {"": "Night classes", "Thursday": "7:00pm","Friday": "7:00pm" },
+        {"": "Team meeting", "Wednesday": "10:00am" },
+        {"": "Morning workout", "Monday": "8:00am", "Tuesday": "8:00am", "Wednesday": "8:00am", "Thursday": "8:00am", "Friday": "8:00am" }
+        ]'>My morning schedule
+      </auro-table>
+      <!-- AURO-GENERATED-CONTENT:END -->
+    </div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/basic.html -->
+
+```html
+<auro-table
+  columnHeaders='["","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]'
+  componentData='[
+  {"": "Dance class", "Monday": "5:00pm","Wednesday": "5:00pm" },
+  {"": "Night classes", "Thursday": "7:00pm","Friday": "7:00pm" },
+  {"": "Team meeting", "Wednesday": "10:00am" },
+  {"": "Morning workout", "Monday": "8:00am", "Tuesday": "8:00am", "Wednesday": "8:00am", "Thursday": "8:00am", "Friday": "8:00am" }
+  ]'>My morning schedule
+</auro-table>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Attribute Examples
+
+#### columnHeaders
+
+The `columnHeaders` attribute is used to provide the column header content for the the table.
+
+</auro-accordion>
+
+#### componentData
+
+The `componentData` attribute is used to provide the table TD content for the table.
+
+#### noWrap
+
+The `nowrap` attribute, when applied, will prevent table content from word wrapping, i.e. all text content will be on a single line.
+
+</auro-accordion>
+
+### Slot Examples
+
+#### default
+
+Default slot content will be displayed when `columnHeaders` and `componentData` are not provided.
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/defaultSlot.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/defaultSlot.html -->
+  <auro-table>
+    My morning schedule
+  </auro-table>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/defaultSlot.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/defaultSlot.html -->
+
+```html
+<auro-table>
+  My morning schedule
+</auro-table>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>

```

Comment:
This was intentional. There isn't a way to demonstrate those API attributes individually since we don't render the table wthout both columnHeaders and copmponentData. And the basic example already shows them.

I wasn't sure what to use for `noWrap` because when I built a demo for it, it just blows out the page and doesn't constrain to within the example wrapper.

Ideas?